css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,319 @@
+Abstract:
+Abstract: New in this level is [[#at-ruledef-sheet]].
+Ignored Terms: auto, flex items, <supports-condition>, <declaration>, scope proximity, encapsulation context
+Informative Classes: ex
+</pre>
+
+<pre class=link-defaults>
+spec:dom; type:dfn;
+	text:shadow tree
+	for:tree; text:root
+	text:shadow root; for:/
+spec:dom; type:dfn; text:parent element
+spec:css-color-4; type:property; text:color
+spec:css-values-3; type: value; text:ex
+spec:css-conditional-3; type:at-rule; text:@media
+spec:mediaqueries-4; type:type; for:@media; text:all
+spec:mediaqueries-4; type:type; text:<media-query>
+spec:selectors-4; type:dfn; text:subject
+spec:selectors-4; type:dfn; text:selector
+spec:selectors-4; type:dfn; text:combinator
+spec:html; type:element; text:style
+spec:css-scoping-1; type:dfn; text:shadow host
+</pre>
+
+<pre class=ignored-specs>
+spec:mediaqueries-5
+spec:css-values-4
+spec:css-fonts-4
+</pre>
+
+<h2 id="intro" oldids='filtering,fragments,stages-examples,actual,used,computed,cascaded,declared,specified,value-stages,all-shorthand,aliasing,shorthand,content-type,import-processing,conditional-import,at-import,defaulting,initial-values,inheriting,defaulting-keywords,initial,inherit,inherit-initial,default'>
+Introduction and Missing Sections</h2>
+
+	Issue: This is a diff spec over <a href="https://www.w3.org/TR/css-cascade-6/">CSS Cascading and Inheritance Level 6</a>.
+	It is currently an Exploratory Working Draft:
+	if you are implementing anything, please use Level 6 as a reference.
+	We will merge the Level 6 text into this draft once it reaches CR.
+
+<!-- Big Text: Stylesheet 
+
+ ███▌  █████▌ █   ▐▌ █▌    █████▌  ███▌  █▌  █▌ █████▌ █████▌ █████▌
+█▌  █▌   █▌   ▐▌  █  █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+█▌       █▌    █ ▐▌  █▌    █▌     █▌     █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌    ▐▌█   █▌    ████    ███▌  █████▌ ████   ████     █▌  
+    █▌   █▌     █▌   █▌    █▌         █▌ █▌  █▌ █▌     █▌       █▌  
+█▌  █▌   █▌     █▌   █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌     █▌   █████ █████▌  ███▌  █▌  █▌ █████▌ █████▌   █▌  
+-->
+
+<h2 id='stylesheet-section'>
+Stylesheet</h2>
+
+	A <dfn export>stylesheet</dfn>
+	is a collection of style rules. 
+	
+	These collections may exist as:
+	<ol>
+		<li>An instance of file with a .css extension and/or MIME type of "text/css" embedded within a document.
+		<li>Collection of CSS rules within a document.
+		<li>CSSOM Stylesheet objects.
+		<li>Any of the above may be subdivided into named stylesheets with the ''@sheet'' keyword.
+	</ol>
+
+	A <dfn>sheet-name</dfn> is a string identifying
+	 inner named sheets within a stylesheet.
+
+	Syntactically, a stylesheet may include an explicit [=sheet-name=], and is represented
+	by the <<sheet-name>> in ''@sheet'' and ''@import'' rules,
+	which is a single <<ident>> token:
+
+	<pre class='prod'>
+		<dfn><<sheet-name>></dfn> = <<ident>>
```

Comment:
```suggestion
		<dfn><<sheet-name>></dfn> = <<custom-ident>> | <<string>>
```

For consistency with [`@keyframes`](https://drafts.csswg.org/css-animations-1/#keyframes)

Reply:
This is interesting - `@layer` doesn't allow the `<<string>>` version, but `@keyframes` does. I filed an issue with the CSSWG for this: https://github.com/w3c/csswg-drafts/issues/12005